core/templates/dev/head/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.directive.html

```diff
@@ -20,6 +20,7 @@ <h3>Review Suggestion</h3>
       </div>
       <div ng-show="suggestionEditorIsShown">
         <ck-editor-4-rte ng-model="suggestionData.newSuggestionHtml"></ck-editor-4-rte>
+        <ck-editor-5-rte ng-model="suggestionData.newSuggestionHtml"></ck-editor-5-rte>
```

Comment:
Don't you need ng-ifs over here?

Reply:
Missed now updated PTAL!

Reply:
Just curious, how was this missed while testing? It would have displayed 2 RTEs right?

Reply:
I Think it will display the last-mentioned RTE